framework/core/js/src/common/utils/isDark.ts

```diff
@@ -8,18 +8,24 @@
  * to preserve design consistency.
  */
 export default function isDark(hexcolor: string): boolean {
+  // return if hexcolor is undefined or shorter than 4 characters, shortest hex form is #333;
+  // decided against regex hex color validation for performance considerations
+  if (!hexcolor || hexcolor.length <= 4) {
```

Comment:
```suggestion
  if (!hexcolor || hexcolor.length < 4) {
```

Reply:
Is `hexcolor` here including the `#` or not? That would make a world of difference.

Reply:
it includes `#`